packages/backend/src/modules/tvl/modules/ChainTvlModule.ts

```diff
@@ -108,9 +109,12 @@ export function chainTvlModule(
       logger,
     )
 
-  const circulatingSupplyTokens = tokens.filter(
-    (t) => t.chainId === config.chainId && t.formula === 'circulatingSupply',
-  )
+  const circulatingSupplyTokens = tokens
+    // temporary solution - will be removed once tvl2 migration is complete
```

Comment:
Please create a task for that

Reply:
This will be removed once we remove tvl v1